src/webviews/apps/home/components/home-app.ts

```diff
@@ -0,0 +1,105 @@
+import { css, html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { repeat } from 'lit/directives/repeat.js';
+import { when } from 'lit/directives/when.js';
+import type { OnboardingState } from '../../../home/protocol';
+import type { OnboardingItemConfiguration } from '../model/gitlens-onboarding';
+import { onboardingConfiguration } from '../model/gitlens-onboarding';
+
+@customElement('gl-home-app')
+export class GlHomeApp extends LitElement {
+	@property({ type: Object })
+	state?: OnboardingState;
+
+	static override readonly styles = css`
+		gl-progress-tracker {
+			padding: 0 20px;
+			--progress-color: var(--vscode-charts-blue);
```

Comment:
We're working to prefix all our web component custom properties with `--gl-`.

Reply:
done